packages/preview/tinyset/0.1.0/lib.typ

```diff
@@ -0,0 +1,31 @@
+/// You must create a header to get the rest of the style
+#let header(number: 1, class: "Math 3551 - Fall 2024", name: "Your Name") = {
+  set text(font: "New Computer Modern Math")
+  align(center)[
+    #class \
+    #name - Homework #number
+  ]
+  set par(justify: true)
+}
+
+// custom shorthands, feel free to comment these out if they are in the way.
```

Comment:
For information it is not possible for package users to edit the contents of a package.